package.json

```diff
@@ -37,17 +37,21 @@
 		"demo": "http-server"
 	},
 	"dependencies": {
-		"vl-ui-core": "^7.1.1"
+		"vl-ui-core": "^7.2.2"
 	},
 	"peerDependencies": {
 		"vl-ui-core": "^7.1.0"
 	},
 	"devDependencies": {
-		"vl-ui-body": "^1.0.7",
-		"vl-ui-button": "^5.0.9",
-		"vl-ui-form-grid": "^3.0.8",
-		"vl-ui-form-message": "^5.0.6",
-		"vl-ui-input-field": "^3.2.7",
-		"vl-ui-util": "^5.3.7"
+		"@govflanders/vl-ui-form-structure": "^3.12.17",
+		"vl-ui-action-group": "^3.1.3",
+		"vl-ui-body": "^1.0.8",
+		"vl-ui-button": "^5.0.10",
+		"vl-ui-form-grid": "^3.0.9",
+		"vl-ui-form-message": "^5.1.0",
+		"vl-ui-icon": "^5.2.2",
+		"vl-ui-input-field": "^3.2.11",
+		"vl-ui-link": "^4.0.10",
+		"vl-ui-util": "^5.3.12"
```

Comment:
is da niet een hele grote lijst om gewoon ne form-group te laten zien?

Reply:
Nee, er wordt een volledig formulier getoond analoog aan demo pagina van Webuniversum.

Reply:
eerder dan "ja, er wordt een volledig formulier getoond analoog aan demo pagina van Webuniversum" zou ik zeggen :-)

Reply:
Ja, het is een heel grote lijst. Nee, er wordt niet alleen een form-group in isolatie getoond.

Reply:
Ja daar verschillen we van mening of dat altijd nodig is denk ik. Maar consistent met de andere.

Reply:
Agree to disagree.